server/agents/Coordinator/agent.py

```
@@ -1,11 +1,11 @@
 from google.adk.agents import LlmAgent
 # from Coordinator.receipt_ingestion.agent import root_agent as receipt_ingestion_agent
-# from Coordinator.spending_insight.agent import root_agent as spending_agent
+from Coordinator.spending_insight.agent import root_agent as spending_agent
```

Ruff (0.12.2)

`Coordinator.spending_insight.agent.root_agent` imported but unused

Remove unused import: `Coordinator.spending_insight.agent.root_agent`

(F401)